CHANGELOG.md

```diff
@@ -25,6 +26,115 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v0.7.0
+
+The v0.7.0 release focuses on refining and stabilizing existing APIs. This
+included a focus on both conformance tests and clarifying ambiguous parts of the
+API spec.
+
+## Features Graduating to Standard
+In addition to those broad focuses, 2 features are graduating to the
+standard channel:
+
+* GEP-1323: Response Header Modifiers (#1905, @robscott)
+* GEP-726: Path Redirects and Rewrites (#1905, @robscott)
+
+## GEPs
+There are a lot of interesting GEPs in the pipeline right now, but in v0.7.0
+we're highlighting two experimental GEPs:
```

Comment:
Curious as to the reasoning for highlighting these specific GEPs?

Reply:
The only ones that meaningfully changed that have status >= experimental, will clarify.

Reply:
Clarified, feel free to reopen if I missed anything.